### PR TITLE
fixes failing smoke tests

### PR DESCRIPTION
**Pull request by entrpn**
This fixes some smoke tests that are failing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ MaxDiffusion started as a fork of [Diffusers](https://github.com/huggingface/dif
 
 Whether you are forking MaxDiffusion for your own needs or intending to contribute back to the community, a full suite of tests can be found in `tests` and `src/maxdiffusion/tests`.
 
-To run unit tests, simply run:
+To run unit tests, you'll need to [install gcsfuse](https://cloud.google.com/storage/docs/cloud-storage-fuse/install) then simply run:
```
Comment (aireenmei) [inline README.md]: Do we recommend users to run setup.sh? It has gcsfuse installation: https://github.com/AI-Hypercomputer/maxdiffusion/blob/main/setup.sh#L28-L32. But I don't see it mentioned in README.md

**Review by aireenmei — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (entrpn) [reply README.md]: Oh that's a good idea. Let me try it first and make sure everything works as expected then I will update the README

Comment (aireenmei) [reply README.md]: We already use it for github unit test: https://github.com/AI-Hypercomputer/maxdiffusion/blob/main/.github/workflows/UnitTests.yml#L43

Comment (entrpn) [reply README.md]: @aireenmei please take another look.

**Review by aireenmei — APPROVED** [review]
LGTM! Thanks Juan!

**Pull request merged**